### PR TITLE
Don't split path name. Fixes #8

### LINKED ISSUE #8: Incorrect filename generated in coverage's JSON
When I run my tests and generate a report, the JSON files generated have the base filename, but are missing the actual path. For instance, the generated name is `nimja.nim` as opposed to `src/nimjapkg/nimja.nim`. When I manually change the filename, everything works fine. It's just an incorrect filename being generated.

### PR DESCRIPTION
**Pull request by ghost**
By not splitting the path name, we avoid the issue described in #8. This commit fixes that issue.

Alternatively, we can check the filename against the current path and only generate a relative path to the file.

### CODE REVIEW

File: nimcoverage.nim [hunk 1]
```diff
@@ -8,4 +8,5 @@ if getEnv("NIM_COVERAGE_DIR").len == 0:
     echo "NIM_COVERAGE_DIR environment variable not set"
     quit 1
 
+echo "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
```
Comment (yglukhov) [inline nimcoverage.nim]: Is this required?

Comment (ghost) [reply nimcoverage.nim]: Debug statement, must've missed it, sorry!

**Comment by ghost** [comment]
Fixed that debug statement. However, there's still the issue of the failing test. We'll need to change that test, but I'd like to get your input on whether or not the code looks good before screwing around with the test.

**Comment by yglukhov** [comment]
Yeah, looks good. But it seems you've added a binary file to the repo :)

**Comment by ghost** [comment]
Sure did! Wow I'm super sorry, that's what I get for coding so late haha. I'll fix everything up and squash as soon as I can.

**Comment by ghost** [comment]
Alright. I had to introduce two new procs to the test file. One of them imports `process.cwd()` from nodejs because that's the only way to get the current working directory in the js backend. The other proc cleans the output of `process.cwd()` because for some reason the string it returns is messed up, and without this `convert()` proc, the test file has a runtime error.

**Comment by yglukhov** [comment]
Thank you!

**Comment by ghost** [comment]
Awesome, thank you very much!

**Pull request merged**